### ISSUE TITLE
Windows defender detects Phonzy.B1ml trojan in the newest release zip 1.72.4

### ISSUE THREAD

**Issue by HenningSte** [Electron-x/GbxDump#3]
As in title.
I'm not versed enough to further investigate the code, so I just wanted to raise the issue if it isn't known yet.
The same happens when I download the version linked on wolfgang rolkes site.

![image](https://github.com/Electron-x/GbxDump/assets/30557436/b351cef0-4183-44a7-bcca-9952a96b6448)

**Comment by Electron-x** [comment]
Thank you for reporting the issue! Normally I have VirusTotal check the archive with the executables before releasing it, and I also test the program for a day or two beforehand. This time I did without both and also didn't do a complete rebuild of the project. However, I don't have an explanation for the many false-positive scan results. Rebuilding the entire project again resulted in the usual scores (3 to 4 false positives for the 32-bit version and 0 to 1 false positive for the 64-bit version of GbxDump v1.72.5). The 32-bit version was rated as malicious by the heuristic analysis of some antivirus programs right from the start. For some time I could counteract this by changing code optimization options. Now only signing the application with a code signing certificate would help.

[Link to gbxdump.zip (v1.72.5) at VirusTotal](https://www.virustotal.com/gui/file/d01180936957f7e16ed3ca9e062ab2d45ef1757463ca523002cdcfa3dc871de1)

For transparency, I would like to take this opportunity to explain why there were revisions 1.72.1 to 1.74.4 with the same change history on the release page.
Since there have been few changes in the header of .gbx files lately, I've been doing a little work on optimizing the program code. However, the routine for displaying the thumbnail could only display 24 bpp images (maps) and 32 bpp images (items). Here I wanted to extend the code to support all possible color depths and to be able to reuse it outside the project. I then got ambitious and wanted to support ICC color profiles and implement the correct copying of such images to the clipboard. To test this functionality it was then necessary that the program can also open JPEG and BMP files. Since all this had nothing to do with GBX files, the previous releases were simply replaced with the same notes.
I'm still unsure whether I'd better move these extensions to a separate project.
